nucliadb/tests/ingest/unit/orm/test_resource.py

```diff
@@ -272,62 +270,3 @@ async def test_apply_fields_calls_update_all_field_ids(txn, storage, kb):
     resource.update_all_field_ids.call_args[1]["deleted"] == [
         FieldID(field_type=FieldType.CONVERSATION, field="to_delete"),
     ]
-
-
-async def test_apply_extracted_vectors_cut_by_dimension(txn, storage, kb):
```

Comment:
Is this test somewhere else? It would be nice to retain it, it's testing something that's not specific of the Brain, but rather ensuring the right thing (cut vectors) is sent to the index.